simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/StationComponent.java

```diff
@@ -59,6 +57,15 @@ public GetVariableResult getVariable(GetVariableDatum getVariableDatum) {
         return accessor.get(new AttributePath(component, variable, attributeType));
     }
 
+    /**
+     *  Retrieves the name of the variables for this specific component.
+     *
+     * @return Set with name of variables
+     */
+    public Set<String> getVariableNames() {
```

Comment:
do we support variable instances like var["blah"]?

Reply:
What do you mean?

Reply:
Ok i saw it. I don't think it is supported right now, ad i did not see anything related to instances in the code